includes/Plugin_Main.php

```diff
@@ -59,8 +60,9 @@ public function add_hooks() {
 			require_once WP_PLUGIN_CHECK_PLUGIN_DIR_PATH . 'cli.php';
 		}
 
+		$admin_ajax = new Admin_AJAX();
```

Comment:
Nit-pick: This doesn't need to be in a variable, we could simply use `new Admin_Page( new Admin_AJAX() )`.

Reply:
I find this more readable personally. I can change it you really hate it.